util/test-hb-subset-parsing.c

```diff
@@ -7,11 +7,8 @@
 static
 hb_face_t* open_font(const char* path)
 {
-  hb_blob_t *blob = hb_blob_create_from_file_or_fail (path);
-  g_assert(blob);
-  hb_face_t* face = hb_face_create(blob, 0);
-  hb_blob_destroy(blob);
-
+  hb_face_t *face = hb_face_create_from_file_or_fail (path, 0);
```

Comment:
I don’t know why this didn’t cause (or show) a memory leak before, but now one bot is failing because the face returned from the second `open_font()` call below is not destroyed: https://app.circleci.com/pipelines/github/harfbuzz/harfbuzz/8892/workflows/42c621fe-708a-4f38-bb8d-71097f0123e2/jobs/224684?invite=true#step-104-71803_110

Reply:
Fixed.